KarhooSDK/Api/Request/TripBooking.swift

```diff
@@ -11,20 +11,17 @@ import Foundation
 public struct TripBooking: KarhooCodableModel, Equatable {
     
     public let quoteId: String
-    public let passengers: Passengers
```

Comment:
Why was Passengers removed from here?

In the documentation, passengers is still passed through this object.
It is also missing:
Loyalty_programme
loyalty_points
train_time
train_number
cost_center_reference
partner_trip_id

Reply:
I will do that in the next ticket. I will add Passengers back then.

Reply:
added the missing ones.